helpers/GenerateResourcesAndImage.ps1

```diff
@@ -240,13 +240,19 @@ Function GenerateResourcesAndImage {
 
     try {
         # Login to Azure subscription
+        try {
+            $azAccount = az account show -o none
```

Comment:
Where `azAccount` variable is used? Why do we need it?
`az account show -o none` always returns nothing because `-o none` means `--output none`. What do we expect to set to `azAccount` variable?